roles/filetree_create/templates/current_credentials.j2

```diff
@@ -3,19 +3,24 @@
 controller_credentials:
 {% endif %}
   - name: "{{ current_credentials_asset_value.name }}"
-    description: "{{ current_credentials_asset_value.description }}"
+    description: "{{ template_overrides_global.credential.description | default(current_credentials_asset_value.description) }}"
     credential_type: "{{ current_credentials_asset_value.summary_fields.credential_type.name }}"
-{% if current_credentials_asset_value.organization is defined and current_credentials_asset_value.organization is not none %}
+{% if template_overrides_global.credential.organization is defined or (current_credentials_asset_value.organization is defined and current_credentials_asset_value.organization is not none) %}
```

Comment:
This is the only place `organization` is allowed to be overriden... and it comes only from `template_overrides_global`... I'm not sure if it makes sense...